iohub/cli/cli.py

```diff
@@ -87,3 +89,60 @@ def convert(input, output, grid_layout, chunks):
         chunks=chunks,
     )
     converter()
+
+
+@click.command()
+@click.help_option("-h", "--help")
+@click.argument(
+    "csvfile",
+    type=click.File("r"),
+)
+@click.argument(
+    "zarrfile",
+    type=click.Path(
+        exists=True,
+        file_okay=True,
+        dir_okay=False,
```

Comment:
```suggestion
        dir_okay=True,
```
A .zarr is a directory, right? I couldn't get it to work without this change. How did you test this?